### PR TITLE
[bitnami/sealed-secrets] fix: 🐛 ServiceMonitor endpoint

### PR DESCRIPTION
**Pull request by aneurinprice**
### Description of the change

Change configures the container and service to work with the ServiceMonitor. By default it will configure the metrics on 8081.

### Benefits

The servicemonitor will actually work.

### Possible drawbacks

n/a

### Applicable issues

n/a

### Additional information

Currently, the serviceMonitor points towards port `8080` on the sealedSecrets service. Metrics are not served on that port.

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by carrodher** [comment]
Thank you for initiating this pull request. We appreciate your effort. Just a friendly reminder that it's important to sign your commits. Adding your signature certifies that you either authored the patch or have the necessary rights to contribute the changes. You can find detailed information on how to do this in the [“Sign your work” section of our contributing guidelines](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work).

Feel free to reach out if you have any questions or need assistance with the signing process.

**Comment by aneurinprice** [comment]
@carrodher Fixed 👍

**Comment by migruiz4** [comment]
Hi @aneurinprice,

I see you included changes to the `bitnami/zookeeper` chart in this PR too.
I'm sorry but we can not accept PRs modifying more than one chart, so those changes would need to be submitted in a separate PR could you please fix that?

**Comment by aneurinprice** [comment]
@migruiz4 My mistake, fixed 👍

**Review by migruiz4 — CHANGES_REQUESTED** [review]
I have noticed these changes are related to https://github.com/bitnami-labs/sealed-secrets/pull/1369.

I'm sorry but I can not accept this PR, as the approach we should follow should be to imitate upstream:
- Add a `metrics` containerPort 8081.
- Create a new `-metrics` service.
- Update the serviceMonitor to use the new metrics service.

Additionally, we should format those changes to follow our chart standards:
- Add control value `metrics.enabled`.
- Add value `containerPorts.metrics`.
- Format the new metrics service and its associated values according to our guidelines ([Example](https://github.com/bitnami/charts/blob/main/bitnami/drupal/templates/metrics-svc.yaml))

I know updating this PR may be too much to ask, so feel free to let me know if you would like to update this PR or close it, in which case I will submit the changes as soon as possible.

**Comment by aneurinprice** [comment]
I'll take care of it.

**Comment by aneurinprice** [comment]
@migruiz4 Ping

File: bitnami/sealed-secrets/Chart.yaml [hunk 1]
```diff
@@ -29,4 +29,4 @@ name: sealed-secrets
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/sealed-secrets
 - https://github.com/bitnami-labs/sealed-secrets
-version: 2.0.2
+version: 2.0.2
```
Comment (migruiz4) [inline bitnami/sealed-secrets/Chart.yaml:32]:
```suggestion
version: 2.1.0
```

**Review by migruiz4 — APPROVED** [review]
Thank you for applying the suggested changes!

**Pull request merged**